### DISCUSSION TITLE
Use with PHP in linux

### DISCUSSION THREAD

**Discussion by C1977** [BieremaBoyzProgramming/bbpPairings#17] [Q&A]
Hello, as I can see this is a exe file. I made a test on windows and it worked very good.
I would like to implement bbpPairing in my PHP program running at German provider Strato with Linux. Here I can't run windows or java programs. Is there any chance?

**Comment by jbierema** [comment]
@C1977 Hi! The [Releases page](https://github.com/BieremaBoyzProgramming/bbpPairings/releases) has Linux versions available. They do have an exe extension, but you should be able to run them on Linux. Or you can rename the file and remove the extension. I hope that helps!
